eta-meeting-organizer-frontend/src/app/shared/components/header-layout.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+import { User } from '~/app/models/placeholder-user.model';
+import { ApiCommunicationService } from '~/app/shared/services/api-communication.service';
+
+@Component({
+  selector: 'app-header',
+  styles: [`
+    .header{
```

Comment:
nagyon sok ez a custom css, a nagy részét tudja kezelni a bootstrap, ha mindenképpen saját szinezést akartok akkor a https://materialtheme.arcsine.dev/-en keresztül csináljatok egy új themet, ha beégettek szineket és később én azt mondom minden gomb legyen rózsaszin akkor kellemetlen lesz minden helyen átirogatni a csst